src/Evolution/Executables/CurvedScalarWave/EvolveCurvedScalarWave.hpp

```diff
@@ -142,6 +167,35 @@ struct EvolutionMetavars {
                                     Frame::Inertial>,
       typename system::gradient_variables>;
 
+  static constexpr bool interpolate = volume_dim == 3;
+  // We want to observe on a spherical shell of radius r so we use a Kerr
```

Comment:
@markscheel do we not have a target for just a spherical surface? If not, should we add one? It seems a bit confusing to use a Kerr horizon just to get a sphere